twint/tweet.py

```diff
@@ -114,6 +114,10 @@ def Tweet(tw, config):
         try:
             ts = translator.translate(t.tweet)
             t.translate = ts.text
+            t.trans_src = ts.src
+            t.trans_dest = ts.dest
         except:
```

Comment:
You should specify the correct exception to handle

Reply:
what do you mean ?

src by default is **en** and dest=**auto**, so the translate method will trigger an exception

ref. https://py-googletrans.readthedocs.io/en/latest/#googletrans.Translator.translate

Reply:
Can you provide an example ? (I started python 1 month ago ^^ with Twint, I am more Golang like)

Reply:
Ok so let's say that you have a dict like `dict1 = {'first_key':'first_value', 'second_key':'second_value'}`. Now if you do something like `third_value = dict1['third_key']`, Python will raise a  `KeyError` exception since in `dict1` the key `third_key` is not defined. To handle this exception you have to do something like

```python
try:
    third_value = dict1['third_key']
except KeyError:
    print("dict1 has not third_key as key")
```

Now if you don't specify `except KeyError`, `except:` will handle every possible exception, like `KeyboardInterrupt` (when you hit `CTRL + C`)

Basically you should check that library and see which exceptions could be raised, and handle that specific one